docs/content/1.guide/6.fetch.md

```diff
@@ -0,0 +1,73 @@
+---
+icon: i-ri-global-line
+---
+
+# Fetch
+
+Nitro provides a built-in fetch API that can be used to get data from server endpoints or from other sources. It's build on top of the [unjs/ofetch](https://ofetch.unjs.io) package.
+
+This is important to use it instead of the built-in Node `fetch` or Axios because it's optimized for serverless environments and agnostic to the runtime.
```

Comment:
$fetch only for convention. we shall only discourage axio

```suggestion
This is important to use fetch-api (`fetch` or `$fetch`) instead of libraries such as axios because it's optimized for serverless environments and agnostic to the runtime.
```